reg_tests/test_files/ActLineSimpleNGP/ActLineSimpleNGP.i

```diff
@@ -40,17 +40,17 @@ linear_solvers:
   type: tpetra
   method: gmres
   preconditioner: sgs
```

Comment:
```suggestion
  preconditioner: mt_sgs
```

@jvo1012 we want to use `mt_sgs` on GPUs. Can you please check and rebless the gold files with this setting?

Reply:
Done